janis_core/tests/tests_util.py

```diff
@@ -82,11 +82,11 @@ def test_4_items(self):
         )
 
 
-class testtranslate(TestCase):
-    def test1(self):
-        from janis_bioinformatics.tools.cutadapt import CutAdapt_2_6
-
-        CutAdapt_2_6().translate("wdl")
+# class testtranslate(TestCase):
+#     def test1(self):
+#         from janis_bioinformatics.tools.cutadapt import CutAdapt_2_6
+#
+#         CutAdapt_2_6().translate("wdl")
```

Comment:
@illusional just want to double check that it's okay to disable this (since there is no assertion). As this require installation of janis_bioinformatics

Reply:
Yep, it's a bad commit from me, when tools don't translate properly, I put them here so I can properly debug what's happening. It's commented out in v0.11.x.